Knoema.Client/Client.cs

```diff
@@ -605,6 +605,11 @@ public async Task<string[]> UnloadToLocalFolder(PivotRequest request, string des
 			return fileNames;
 		}
 
+		public Task<RawDataFilesModel> GetRawDataFiles(string datasetId)
+		{
+			return ApiGet<RawDataFilesModel>($"/api/1.0/meta/dataset/{datasetId}/raw");
```

Comment:
I don't get why it's called raw? data/raw is streaming API. And we had a thing called dataset image for source files. Is this different?

Reply:
it was projected on the Knoema side